shell/platform/android/test/io/flutter/embedding/engine/systemchannels/PlatformChannelTest.java

```diff
@@ -113,6 +115,61 @@ public void setSystemExclusionRectsSendsJSONExceptionOnIncorrectDataShape() thro
         );
     }
 
+    @Test
+    public void getSystemExclusionRectsSendsExclusionRectsToFrameworkOnSuccess() throws JSONException {
+        DartExecutor dartExecutor = mock(DartExecutor.class);
+        PlatformChannel platformChannel = new PlatformChannel(dartExecutor);
+        PlatformMessageHandler platformMessageHandler = mock(PlatformMessageHandler.class);
+        platformChannel.setPlatformMessageHandler(platformMessageHandler);
+
+        int top = 0;
+        int right = 500;
+        int bottom = 250;
+        int left = 0;
+
+        ArrayList<Rect> expectedExclusionRects = new ArrayList<Rect>();
+        Rect gestureRect = new Rect(left, top, right, bottom);
+        expectedExclusionRects.add(gestureRect);
+        when(platformMessageHandler.getSystemGestureExclusionRects()).thenReturn(expectedExclusionRects);
+
+        MethodCall callGetSystemGestureExclusionRects = new MethodCall(
+            "SystemGestures.getSystemGestureExclusionRects",
+            null
+        );
+        ResultsMock resultsMock = mock(ResultsMock.class);
```

Comment:
I have the same question here about `ResultsMock` as the other PR. Do we need to define our own class if we're using `mock()`? Can Mockito just `mock()` the interface, instead?

Reply:
Yeah, I had a misunderstanding of how `Mockito` worked when I wrote the original tests. I've removed them in https://github.com/flutter/engine/pull/11804